docs/11-reports/MULTI_TENANCY_FINAL_15_PERCENT_COMPLETE_2025-11-15.md

```
@@ -0,0 +1,899 @@
+
+# Test tenant (no provisioning)
+python scripts/tenant/provision_tenant.py \
+    --name "Test Org" \
+    --slug "test-org" \
+    --admin-email "test@example.com" \
+    --skip-provision \
+    --verbose
+```
+
+**Sample Output:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)